mkdocs.yml

```diff
@@ -27,9 +27,10 @@ nav:
       - 'Other file formats': 'data_providers/data_format/other_formats.md'
       - 'Data format FAQs': 'data_providers/data_format/faqs.md'
   - 'Command line tools':
-    - 'overview': 'command_line_tools/overview.md'
+    - 'Overview': 'command_line_tools/overview.md'
     - 'safedata_validate': 'command_line_tools/safedata_validate.md'
     - 'safedata_zenodo': 'command_line_tools/safedata_zenodo.md'
+    - 'safedata_server': 'command_line_tools/safedata_server.md'
```

Comment:
Has this markdown file been added yet, I can't see it in the list of files changed